health/notifications/ntfy/metadata.yaml

```diff
@@ -53,6 +53,34 @@
               role_recipients_ntfy[proxyadmin]="https://SERVER5/TOPIC5"
               role_recipients_ntfy[sitemgr]="https://SERVER6/TOPIC6"
               ```
+          - name: 'NTFY_USERNAME'
+            default_value: ''
+            description: "The username for netdata to use to authenticate with an ntfy server."
+            required: false
+            detailed_description: |
+            This should be used with `NTFY_PASSWORD` and is only useful on self hosted instances of ntfy. See https://docs.ntfy.sh/config/#users-and-roles for details.
+
+            Ensure that your user has proper read/write access to the provided topic in `DEFAULT_RECIPIENT_NTFY`
+            "
```

Comment:
```suggestion
            Ensure that your user has proper read/write access to the provided topic in `DEFAULT_RECIPIENT_NTFY`.
```

Reply:
Not resolved. I think we need to remove `"` at the end.

Reply:
Oh good catch! I only saw the `.` missing. I will go through the changes quick and see if I missed that on the rest

Reply:
It looks like the issue actually _was_ the trailing quotes. I guess starting strings with `|` means you don't need to end them with anything. At least that is how its done in the rest of the yaml. I removed the trailing `"` from each of the three multi-line strings I added and the language server _seems_ to be fine with it.